### PR TITLE
[keymap] miryoku layout

### PR DESCRIPTION
**Pull request by manna-harbour**
## Description
The miryoku layout is an ergonomic, minimal, orthogonal layout for ergo or ortho keyboards.  The layout is maintained in emacs org-mode tables and converted to QMK keymap data structures using embedded python scripts. The layout is mapped onto keyboards with different physical layouts as a subset without code duplication using the QMK userland feature and C macros.

## Types of Changes

- [x] Keymap/layout/userspace (addition or update)

## Checklist

- [x] My code follows the code style of this project.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: users/manna-harbour_miryoku/miryoku.org [hunk 1]
```diff
@@ -0,0 +1,606 @@
...
+
+*** config.h
+
+Config options.  Automatically included.
+
+[[./config.h][users/manna-harbour_miryoku/config.h]]
+#+BEGIN_SRC C :noweb yes :tangle config.h
+
+// <<header>>
+
```
Comment (zvecr) [inline users/manna-harbour_miryoku/miryoku.org]: The generation should add a header guard
```suggestion

#pramma once
```

Comment (manna-harbour) [reply users/manna-harbour_miryoku/miryoku.org]: Added header guard to config.h. Thank you for the suggestion.

**Comment by fauxpark** [comment]
@manna-harbour the README symlink makes Git on Windows very sad :( Could you please get rid of it if possible?

```
From https://github.com/qmk/qmk_firmware
 * branch                master     -> FETCH_HEAD
Updating 2636464b8..01fb06af6
error: unable to create symlink users/manna-harbour_miryoku/README.org: No such file or directory
```

**Comment by manna-harbour** [comment]
@fauxpark: You should be able to fix that with git config core.symlinks false, but I'm happy to delete the link.

**Comment by fauxpark** [comment]
Yeah, I do know that there is some kind of support for symlinks on Windows, however it looks like it's not enabled OOB, and I think there is an (undocumented) rule against symlinks for exactly this reason.

**Pull request merged**